www/source/tutorials/build-your-own/connect-nginx.html.slim

```diff
@@ -0,0 +1,198 @@
+---
+title: Habitat - Connect to Nginx
+---
+
+section
+ h1 Connect to Nginx
+
+ p If you have successfully completed the sample app tutorial, you will see that the application server Puma was used to serve out the index page when you visited the local site through your Web browser. For production use, it would be much better to use a web server like Nginx as a reverse proxy to buffer requests and responses between users and your Rails application.
```

Comment:
For production use, you should add a web server like Nginx as a reverse proxy to buffer requests and responses between users and your application.

Suggestion: this content does not have to be limited to Rails scaffolding. It is generic and applicable to all applications, with or without a Scaffolding.

Also, this will need to be updated when we have composite plans out.

Reply:
+1 to your suggestion. We can use #1559 to track this as a more generic topic.